pennylane/decorator.py

```diff
@@ -83,30 +91,20 @@ def qfunc1(x):
         qnode1 = qml.QNode(qfunc1, dev1)
         result = qnode1(0.543)
 
-.. raw:: html
 
-    <h3>Classical interface support (experimental)</h3>
+Machine learning interfaces
```

Comment:
```suggestion
Using QNodes with machine learning interfaces
```

Reply:
This might not render properly in the TOC due to the length; can we shorten it?

Reply:
Oh I see. I guess it's ok if the first sentence of this section points it out then...ignore this one